content/sensu-core/2.0/reference/silencing.md

```diff
@@ -0,0 +1,196 @@
+---
+title: "Silencing"
+description: "The silencing reference guide."
+weight: 1
+version: "2.0"
+product: "Sensu Core"
+platformContent: true
+menu:
+  sensu-core-2.0:
+    parent: reference
+---
+
+# Silencing
+
+## How does silencing work?
+Silencing entries are created on an ad-hoc basis via `sensuctl`. When silencing entries are successfully created, they are assigned an `ID` in the format `$SUBSCRIPTION:$CHECK`, where `$SUBSCRIPTION` is the name of a Sensu agent subscription and `$CHECK` is the name of a Sensu check. Silencing entries can be used to silence checks on specific agents by by taking advantage of per-agent subscriptions, e.g. `agent:$AGENT_NAME`.
```

Comment:
Additionally, I was thinking maybe we should replace `Sensu agent` with `Sensu entity` since it's the agent's entity that subscribes. What do you think?

Reply:
I agree, it makes more sense to refer to entities over agents. I'm already using that terminology other places in the doc, so I'll update it to be consistent.